pokemongo_bot/step_walker.py

```diff
@@ -19,17 +19,17 @@ def __init__(self, bot, dest_lat, dest_lng):
             dest_lat,
             dest_lng
         )
-
-        self.speed = self.bot.config.walk_max - random() * (self.bot.config.walk_max - self.bot.config.walk_min)
+            
+        self.speed = uniform(self.bot.config.walk_min, self.bot.config.walk_max)
 
         self.destLat = dest_lat
         self.destLng = dest_lng
         self.totalDist = max(1, self.dist)
 
         if self.speed == 0:
-            self.steps = 1
-        else:
-            self.steps = (self.dist + 0.0) / (self.speed + 0.0)
+            self.speed = uniform(3,5)
```

Comment:
@BriceSD Rest assure that I fully understand your point, having a default is always good.

On top of removing teleport, he also wants to bump up any calculated walking speed of 0 to 4.16 ( #4025 ), which is exactly what you don't want. I think we should have more suggestions before I make more commits. @TheSavior @mjmadsen @douglascamata 

I'll be happy to make the edits after we reach an agreement as there seem to be some discrepancy in this matter.

On another slightly related matter, to make the bot stand still, I think a very clear way is to not have any Move or Follow tasks. Allowing the bot to move during CatchPokemon task doesn't sound right to me at all. What do you think?